.github/workflows/ci.yml

```diff
@@ -68,6 +68,10 @@ jobs:
         pip list
         sudo apt-get update
         sudo apt-get -qq install pandoc
+    - name: Check docstrings
+      run: |
+        # Group 1 is related to docstrings
+        pydocstyle --select D1 src/pyhf/pdf.py
```

Comment:
```suggestion
        pydocstyle --select D107 src/pyhf/pdf.py
```

Reply:
@kratsg Why do you only want to run [107 instead of all of group 1](http://www.pydocstyle.org/en/latest/error_codes.html)?

Reply:
I thought you only wanted to fix the issue related to missing docstrings in `__init__`. If we want it to pass only on pdf.py -- then should make sure we have no missing docstrings there?

Reply:
> If we want it to pass only on pdf.py -- then should make sure we have no missing docstrings there?

Yes. `D1` checks all of the error codes in the 1 group. So as `pdf.py` passes `D100` through `D107` then why not run with that?

Reply:
Oh, I misunderstood.